otelcol/collector.go

```diff
@@ -1,8 +1,11 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-// Package service handles the command-line, configuration, and runs the
-// OpenTelemetry Collector.
+// Package otelcol handles the command-line, configuration, and runs the OpenTelemetry Collector.
+// otelcol/collector.go contains the main Collector struct and its constructor [NewCollector].
+// [Collector.Run] starts the collector blocks until it shuts down.
+// [Collector.setupConfigurationComponents] is the "main" function responsible for startup - it orchestrates the loading of the
```

Comment:
A link to a private function will not work here.
```suggestion
// Collector.setupConfigurationComponents is the "main" function responsible for startup - it orchestrates the loading of the
```

Reply:
:/ wasn't sure if this wasn't working.  was hoping it was goland not working.  Sad to hear its just like this